### PR TITLE
Feature manual 76

### PR DESCRIPTION
**Pull request by sanchitanec**
NGSI-LD manual update

### CODE REVIEW

File: doc/en/source/api.rst [hunk 1]
```diff
@@ -1143,6 +1143,977 @@ Here is the Javascript-based code example to terminate a service topology by del
...
 
+
+.. figure:: figures/ngsild_architecture.png
+
+
+
+
+Entities API
+------------
+For the purpose of interaction with Fogflow, IOT devices approaches broker with entity creation request where it is expanded as per given context. Broker further forwards the registration request to Fogflow Discovery in correspondence to the created entity.
```
Comment (naveennec) [inline doc/en/source/api.rst:1160]: [minor]better to write understood or resolved as per given context.

File: doc/en/source/system_overview.rst [hunk 2]
```diff
@@ -111,6 +111,16 @@ to enable FogFlow Ecosystem to interact with Scorpio context broker. The NGSI-LD
 .. _`Integrate FogFlow with Scorpio Broker`: https://fogflow.readthedocs.io/en/latest/scorpioIntegration.html
 
 
+**FogFlow NGSI-LD Support** FogFlow is providing NGSI-LD API support along with NGSI9 and NGSI10. 
+NGSI-LD format aims at utilizing the **linked data model used by Fiware** for the purpose of deriving a hassle free and better mode of communication among components of fogflow or other GE's. It reduces the complexity of maintaining the data among NGSIv1 and NGSIv2 model by establishing a relationship between data to deduce information in a more efficient manner.
```
Comment (naveennec) [inline doc/en/source/system_overview.rst:115]: [minor]NGSI-LD feature like unambiguous and better understanding of context sharing need to be added.

File: doc/en/source/system_overview.rst [hunk 3]
```diff
@@ -111,6 +111,16 @@ to enable FogFlow Ecosystem to interact with Scorpio context broker. The NGSI-LD
...
+
+- The reason for incorporating this model is the dire need of linked data association that are forming the backbone of Edge Computing.
```
Comment (naveennec) [inline doc/en/source/system_overview.rst:117]: [minor]spell mistake "dire"  as direct.

**Review by naveennec — CHANGES_REQUESTED** [review]

File: doc/en/source/system_overview.rst (hunk 2 above)
Comment (sanchitanec) [reply doc/en/source/system_overview.rst:115]: done

File: doc/en/source/system_overview.rst (hunk 3 above)
Comment (sanchitanec) [reply doc/en/source/system_overview.rst:117]: done

File: doc/en/source/api.rst (hunk 1 above)
Comment (sanchitanec) [reply doc/en/source/api.rst]: done

**Pull request merged**